cmake/O2Utils.cmake

```diff
@@ -320,14 +321,55 @@ function(O2_GENERATE_EXECUTABLE)
       MODULE_LIBRARY_NAME ${PARSED_ARGS_MODULE_LIBRARY_NAME}
   )
 
-  ############### install the executable #################
-  install(TARGETS ${PARSED_ARGS_EXE_NAME} DESTINATION bin)
+  if (NOT ${PARSED_ARGS_INSTALL} OR ${PARSED_ARGS_INSTALL})
```

Comment:
Good catch. It is a mistake. It is needed by GENERATE_TEST to tell GENERATE_EXECUTABLE not to install the tests. I will fix it.